stargate-core/app/com/salesforce/mce/stargate/controllers/McSsoController.scala

```diff
@@ -52,8 +51,16 @@ class McSsoController @Inject() (
       id <- request.session.data.get("id")
     } yield (mid.toLong, userId.toLong, id)
 
-    data.fold(Future.successful(Ok.withNewSession)) { case (mid, userId, id) =>
-      sessionTrackingService.destroy(mid, userId, id).map(_ => Ok.withNewSession)
+    val config = SessionConfiguration()
```

Comment:
In this case, you are constructing a new `SessionConfiguration` which actually does not read the config and are not consistent with the existing one, you should use the one provided (injected) by playframework.